### DISCUSSION TITLE
Using STARsolo for bulk data

### DISCUSSION THREAD

**Discussion by jmintch** [alexdobin/STAR#1512] [Q&A]
Thanks a lot for developing this great tool @alexdobin!

Our lab recently generated bulk libraries that both a UMI and a CELseq2-style sample barcode. I was wondering if STARsolo could also be used to map and count transcripts of bulk data that contains UMIs? Would there be any assumptions in the code that would vote against that? (I tested using the sample barcode as a pseudo "cell barcode" and it worked quite nicely, though I am not quite sure if that makes sense)

I'd appreciate any comments.

**Comment by alexdobin** [comment]
Hi @jmintch 

there should be no issue mapping bulk data, since the mapping mapping algorithms is the same.
As long as your barcoding is compatible with options presently implemented in STARsolo, it should work fine.
